aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -189,6 +208,20 @@ void initialize(
     this.warehousePath = Strings.isNullOrEmpty(path) ? null : LocationUtil.stripTrailingSlash(path);
     this.glue = client;
     this.lockManager = lock;
+    this.extensionsEnabled =
+        PropertyUtil.propertyAsBoolean(
+            catalogProperties, GlueExtensionsProperties.GLUE_EXTENSIONS_ENABLED, true);
```

Comment:
It looks like you're defaulting the extensions to be enabled, but does this require the glue bundle/runtime to work?  Why do we need to disable it for all of the tests?